numpy/lib/function_base.py

```diff
@@ -1320,6 +1320,477 @@ def gradient(f, *varargs, **kwargs):
     else:
         return outvals
 
+def divergence(v,*varargs,**kwargs):
```

Comment:
Hello seberg,
You are right, it makes more sense to call gradient as needed. I thought of this but for some reason I also thought that each function should be "stand alone" (which I didn't do in laplace!). It will certainly be easier to understand (for me too!) in this way. I will work on keeping what is new, and have the rest substituted for the appropriate calls to gradient. Yes, all the varargs are uncofortable to me too, In this way (calling gradient), this "freedom" will be removed when calling divergence, curl and laplacian, since they not make much sense for these functions. 
Thanks for your suggestions